### ISSUE TITLE
Roku Card not showing in Lovelace

### ISSUE THREAD

**Issue by Sarai91** [iantrich/roku-card#82]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
1.2.4

**Last working release (if known):**
N/A

**Browser and Operating System:**
Tested on Windows 10 and MacOS Big Sur (Firefox 100.0)

**Description of problem:**
The card does not appear in the Lovelace gui. Attempting to manually add the card in the yaml editor returns the error: `Custom element doesn't exist: roku-card.`

I also attempted to replace the manually added reference in the Settings > Dashboards > Resources menu (`/hacsfiles/roku-card/roku-card.js?hacstag=164367214124`, Javascript Module) with `/local/roku-card/roku-card.js` but that also produced nothing.

**Javascript errors shown in the web inspector (if applicable):**
N/A

**Additional information:**
This was the first card I attempted to add using HACS. I added two other cards successfully, which do show up in the Lovelace GUI (Mini Media Player and RGB Light Card)

**Comment by JPDucky** [comment]
I can also confirm that it is not showing up in Lovelace, nor does it work trying to add it manually

**Comment by JPDucky** [comment]
I went through the process of a new install only to get the above error, and lo and behold, having forgotten ever troubleshooting this, that I should find my own comment lol. 

tldr; still doesn't work.

**Comment by cornfeedhobo** [comment]
@iantrich any hope here? not a js dev and don't have the time to dig in atm, sorry in advance.

**Comment by nextcode4u** [comment]
Use the manual option when  you add a card. Scroll to the bottom. 
You'll have to manually define it with the example on the readme. 
 If you have volume issues try changing the the service to this 
volume_up:
  tap_action:
    action: call-service
    service: media_player.volume_up
    service_data:
      entity_id: media_player.obeythewalrus_tv
volume_down:
  double_tap_action:
    action: call-service
    service: media_player.volume_down
    service_data:
      entity_id: media_player.obeythewalrus_tv

**Comment by rosinscj** [comment]
same issue, cannot find the card once installed via HACS. I don't have this issue w/ other cards.
Manually adding the card did work.
